app/src/main/java/io/heckel/ntfy/util/Util.kt

```diff
@@ -469,3 +470,10 @@ fun copyToClipboard(context: Context, notification: Notification) {
         .makeText(context, context.getString(R.string.detail_copied_to_clipboard_message), Toast.LENGTH_LONG)
         .show()
 }
+
+fun stringToHash(s: String): String {
+    val bytes = s.toByteArray();
+    val md = MessageDigest.getInstance("SHA-256")
+    val digest = md.digest(bytes)
+    return digest.fold("") { str, it -> str + "%02x".format(it) }
+}
```

Comment:
Nit: This could be
```
fun String.sha256(): String {
    val md = MessageDigest.getInstance("SHA-256")
    val digest = md.digest(this.toByteArray())
    return digest.fold("") { str, it -> str + "%02x".format(it) }
}
```

So it can be used as `"myString".sha256()`